torch/_functorch/aot_autograd.py

```diff
@@ -423,7 +420,7 @@
 @dynamo_timed
 def create_aot_dispatcher_function(
```

Comment:
QQ: this name seems a bit odd: it doesn't only create an aot_dispatcher function, it chooses one and actively runs it, returning the compiled fn. If anything this thing is just `aot_dispatch_and_compile`, right?